### PR TITLE
[Serializer] Fix a bug when using groups together with a name converter

### PR DESCRIPTION
**Pull request by dunglas**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |
- Fix the bug
- Increase test coverage

### CODE REVIEW

File: src/Symfony/Component/Serializer/Tests/Normalizer/GetSetMethodNormalizerTest.php [hunk 1]
```diff
@@ -262,6 +273,48 @@ public function testGroupsDenormalize()
...
+        $obj->setSymfony('@coopTilleuls');
+        $obj->setCoopTilleuls('les-tilleuls.coop');
+
+        $this->assertEquals(
+            array(
+                'bar' => null,
+                'foo_bar' => '@dunglas',
+                'symfony' => '@coopTilleuls',
+            ),
+            $this->normalizer->normalize($obj, null, array('groups' => ['name_converter']))
```
Comment (xabbuh) [inline src/Symfony/Component/Serializer/Tests/Normalizer/GetSetMethodNormalizerTest.php]: The short array notation is not supported by PHP 5.3.

**Comment by xabbuh** [comment]
The failing tests in the Translation component don't seem to be related to your changes, but a rebase might fix that.

**Comment by dunglas** [comment]
Removed the short array notation (it became a reflex) and rebased.

**Comment by dunglas** [comment]
ping @symfony/deciders

**Comment by dunglas** [comment]
ping @symfony/deciders. I'll want to include this fix in the last release.

**Comment by fabpot** [comment]
Thank you @dunglas.

**Pull request closed without merging**